build/MacOSXBundleInfo.plist.in

```diff
@@ -265,6 +265,24 @@
 			<key>NSPersistentStoreTypeKey</key>
 			<string>XML</string>
 		</dict>
+		<dict>
+			<key>CFBundleTypeName</key>
+			<string>Compressed MusicXML File</string>
+			<key>CFBundleTypeExtensions</key>
+			<array>
+				<string>mxml</string>
```

Comment:
Same here, but this time I'm more certain about it: simply add mxml as an alternative extension to the existing "Compressed MusicXML File" type, i.e.
```xml
			<key>CFBundleTypeExtensions</key>
			<array>
				<string>mxl</string>
				<string>mxml</string>
			</array>
```
(We could "unify" other file types with multiple extensions too, but I'm not sure whether that should be done in this PR.)

Reply:
At least .xml and .musicxml could be done in this PR

Reply:
Done